week02/Largest_Common_Prefixr.cpp

```diff
@@ -0,0 +1,29 @@
+#Time Complexity:O(nlog(n)) + O(n)
+#Space Complexity:O(1)
```

Comment:
Not optimal. You need to mention what is `n` here. And sorting was completely unnecessary in this problem. The optimal solution takes `O(n)` time, where `n` = number of total characters in all strings.